lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -73,145 +48,112 @@ def fragment_cache(cached_hash, non_cached_hash)
           ActiveModel::Serializer::Adapter::JsonApi::FragmentCache.new.fragment_cache(root, cached_hash, non_cached_hash)
         end
 
-        protected
-
-        attr_reader :fieldset
-
         private
 
-        def serializable_hash_for_collection(options)
-          hash = { data: [] }
-          included = []
-          serializer.each do |s|
-            result = self.class.new(s, instance_options.merge(fieldset: fieldset)).serializable_hash(options)
-            hash[:data] << result[:data]
-            next unless result[:included]
-
-            included |= result[:included]
-          end
-
-          included.delete_if { |resource| hash[:data].include?(resource) }
-          hash[:included] = included if included.any?
-
-          if serializer.paginated?
-            hash[:links] ||= {}
-            hash[:links].update(pagination_links_for(serializer, options))
+        # Build the requested resource objects.
+        # @return [Array] [primary, included] Pair of arrays containing primary and included
+        #   resources objects respectively.
+        #
+        # @api private
```

Comment:
no need to tag as `@api private` in the private scope, imho.  I like the brief comments that say what goes in and what comes out :)